packages/hl7v2-server/src/logger.ts

```diff
@@ -0,0 +1,19 @@
+export interface Logger {
+  info(message: string, ...args: unknown[]): void;
+  error(message: string, ...args: unknown[]): void;
+  debug(message: string, ...args: unknown[]): void;
+}
+
+export class ConsoleLogger implements Logger {
```

Comment:
Can't we use our `out().log`? It includes the requestId on logs, which is super valuable when using logs to debug the behavior of the server.

Reply:
Yep, TIL! Didn't realize we had a logger, ty 😁

Reply:
addressed